docs/migration.md

```diff
@@ -0,0 +1,128 @@
+# Migrate from PostgreSQL Community to Percona Distribution for PostgreSQL 
+
+
+Percona Distribution for PostgreSQL provides the PostgreSQL database and the tools, add-ons, and extensions in a package that is guaranteed to work together and is easy to deploy.
```

Comment:
It's not really a "package" in the classical sense, how about this?
```suggestion
Percona Distribution for PostgreSQL provides the PostgreSQL database and the tools, add-ons, and extensions in a collection that is guaranteed to work together and is easy to deploy.
```

Reply:
Fixed